File: LeetCode/Python Solutions/Excel Sheet Column Title/excel.py
class Solution:
    def convertToTitle(self, columnNumber: int) -> str:
        result = []
        while columnNumber > 0:
            columnNumber -= 1
            result.append(chr(columnNumber % 26 + ord('A')))
            columnNumber //= 26
        return ''.join(reversed(result))



# Checking in terminal/console:
if __name__ == '__main__':
    Sol = Solution()
    Solve = Sol.convertToTitle(columnNumber=701)
    # columnNumber = 701 -> "ZY"
    # columnNumber = 28 -> "AB"
    print(Solve)

File: LeetCode/Java Solutions/Excel Sheet Column Title/excel.java
public class Solution {
    public String convertToTitle(int columnNumber) {
        StringBuilder result = new StringBuilder();
        while (columnNumber > 0) {
            columnNumber -= 1;
            char ch = (char)(columnNumber % 26 + 'A');
            result.insert(0, ch);
            columnNumber /= 26;
        }
        return result.toString();
    }

    public static void main(String[] args) {
        Solution s = new Solution();
        System.out.println(s.convertToTitle(28));  // Outputs AB
    }
}
